test.js

```diff
@@ -1,36 +1,59 @@
 'use strict';
-var assert = require('assert');
+var assert = require('chai').assert;
```

Comment:
Please ignore the 5 failures. Apparently Artifactory (a registry platform) doesn't support the required API end-points for some tests. Removing Artifactory from my setup.

I now have the same failure as y'all have encountered, except that I get the same messages as @lzilioli 

Still looking into it.